src/main/java/com/example/demo/model/Movie.java

```diff
@@ -0,0 +1,133 @@
+package com.example.demo.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Entity
+@Table(name="movies")
+public class Movie {
+    @Id
+    //@GeneratedValue(strategy= GenerationType.AUTO)
+    @Column(name = "movie_id")
+    private Integer movieId;
+
+    @Column(name = "movie_title")
+    private String movieTitle;
+
+    @Column(name = "release_date")
+    private String releaseDate;
+
+    @Column(name = "video_release_date")
+    private String videoReleaseDate;
```

Comment:
Both fields were included in the schema provided, but I couldn't find any further information in the documentation provided in the [original repository](https://git.lasige.di.fc.ul.pt/mmsilva/usbcf-paper/-/tree/2b164ab3e08099370da148607927b35011f16d49/Datasets/ml-100k). The values in the tables are randomly generated, so I was unable to infer anything from there either.

As per the requirements of the assignment, the videoReleaseDate field hasn't been used anywhere in the business logic. So, I've removed it from the class to avoid unecessary ambiguity.

Reply:
Sound good.